components/browser/thumbnails/build.gradle

```diff
@@ -26,18 +26,32 @@ android {
     }
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions.freeCompilerArgs += [
+            "-Xuse-experimental=kotlinx.coroutines.ExperimentalCoroutinesApi"
+    ]
+}
+
 dependencies {
     implementation project(':browser-session')
```

Comment:
You can remove the dependency on `browser-session` now. There are browser session imports in `BrowserThumbnails` but that's only because the KDocs need an update :).

Reply:
Fixed the docs as well.